modular_skyrat/modules/loadouts/loadout_items/under/donator.dm

```diff
@@ -9,10 +9,6 @@
 	name = "Enclave - Officer"
 	item_path = /obj/item/clothing/under/enclaveo
 
-/datum/loadout_item/under/jumpsuit/donator/fashionableskirt
-	name = "Fashionable Skirt"
-	item_path = /obj/item/clothing/under/rank/civilian/skirt
-
```

Comment:
this was just a rip of the bottom-half of another donator item, and had no top. It was added in a PR that, I kid you not, made damaged clothes expose your breasts.
Its never been used, its not a sprite that should have been used, and there's better alternatives if you want toplessness.